main.go

```diff
@@ -212,6 +236,10 @@ func initWidgets() {
 		wg.Done()
 	}()
 	if !minimal {
+		go func() {
```

Comment:
I think we should change this so that the battery is only created if the battery flag is set. The widget count variable also has to be changed to account for this.

Reply:
:sheepishly: yeah.  I got it working, then saw that and made changes I thought would do what you're suggesting and couldn't get it working again.  Lazily, I reverted back to what was working and pushed the pull request.

I'll work on this a bit more.